manifests/broker.pp

```diff
@@ -0,0 +1,20 @@
+# Set up the broker
+class pulp::broker {
+  if $pulp::manage_broker {
```

Comment:
I think I took it too far with the above comment. A lot of puppet modules install those dependencies (foreman most notably) and the database should be manageable by pulp.

I was thinking about pidfilepath and logpath parameters that should not be managed by pulp. It should just install the db with default options and populate it.